marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/flow/setup.xqy

```diff
@@ -5,4 +5,23 @@ hub-test:reset-hub();
 xquery version "1.0-ml";
 import module namespace hub-test = "http://marklogic.com/data-hub/test" at "/test/data-hub-test-helper.xqy";
 import module namespace test = "http://marklogic.com/test" at "/test/test-helper.xqy";
-hub-test:load-artifacts($test:__CALLER_FILE__)
+hub-test:load-artifacts($test:__CALLER_FILE__);
+
+xquery version "1.0-ml";
+
+import module namespace test = "http://marklogic.com/test" at "/test/test-helper.xqy";
+
+xdmp:invoke-function(function() {
```

Comment:
Please build this into the test plumbing itself - the concept of loading static job documents for a test will likely be useful for many other unit tests. The code should go into data-hub-test-helper.xqy - make a "load-jobs" function that loads anything under the "jobs" folder (similar to "content", but written to the jobs data). Then this test doesn't need to do anything other than stick those two job documents under "jobs". 

To account for "batch" documents at some point, it may be worth supporting "test-data/job-data/jobs" and "test-data/job-data/batches".

Reply:
I have made changes so that job and batch docs will be written to jobs db if they are added to test-data/jobs, test-data/batches respectively